mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -5124,88 +5124,9 @@ def test_assign_view_get_request_redirects_to_resolve_check_page(self):
 
             self.assertRedirects(response, reverse('security:resolve_check', kwargs={'check_id': check_id}))
 
-    def test_assign_view_get_request_redirects_to_resolve_check_page_with_kwarg(self):
```

Comment:
Why are you removing tests as part of design system upgrade PR??

Reply:
it's a duplicate. it's line-by-line identical to the one above.